en/3-meeds.md

```diff
@@ -1,80 +1,81 @@
-# 3. Meeds token – the currency of the WoM
+# 3. MEED token – the currency of the WoM
 
-The WoM is powered by its currency – the Meeds (MEED) token. Meeds are used to reward users (through engagement programs), purchase in stores, get Deeds, get financial returns on investments etc.
+Its currency powers the WoM – the Meeds (MEED) token. Users are rewarded with MEED tokens (through engagement programs), purchase in stores, get Deeds, get financial returns on investments, and more.
 
 ## 3.1. Tokenomics
 
-The Meeds **maximum supply is limited to 100 million tokens**, minted progressively. **Meeds are minted at the rate of 10 MEED per minute**. Therefore, the full minting horizon is close to 20 years (as around $10*60*24*365 = 5M$ Meeds are minted per year).
+The **maximum supply of MEED is limited to 100 million tokens**, minted progressively. 
 
-5 Million Meeds (5% of the total supply) were pre-minted initially. Of which, 4.5M have been locked in a vault contract with a linear vesting over 4 years.
-The pre-minted amount was mainly used to acquire software IP, pay for token emission’s legal fees, and host services. 
+The Meeds Association has minted 5 Million MEED (5% of the total supply) at the start to acquire software IP from the funding members, pay for token emission legal fees, and hosting services. 
```

Comment:
Meeds is used mainly to describe the organization while MEED is used for the token.

Reply:
Yes. In this sentence, the Meeds Association term was used on purpose to clarify who minted the first tokens.